tests/network/vmi_istio.go

```diff
@@ -125,6 +126,59 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 			By("Waiting for VMI to be ready")
 			tests.WaitUntilVMIReady(vmi, console.LoginToCirros)
 		})
+		Describe("Live Migration", func() {
+			BeforeEach(func() {
+				tests.SkipIfMigrationIsNotPossible()
+			})
+			migrationCompleted := func(migration *v1.VirtualMachineInstanceMigration) (bool, error) {
+				migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(migration.Name, &metav1.GetOptions{})
+				if err != nil {
+					return false, err
+				}
+				if migration.Status.Phase == v1.MigrationSucceeded {
+					return true, nil
+				}
+				return false, fmt.Errorf("migration is in phase %s", migration.Status.Phase)
+			}
+			allContainersCompleted := func(podName string) (bool, error) {
```

Comment:
can't you polish the func signature a little and just return an error ? i.e. not having all the containers in terminated state would be yet another error.

Reply:
Done